### PR TITLE
Email and retrieve file

### PR DESCRIPTION
**Pull request by fvanderbiest**
please review

### CODE REVIEW

File: celery/extractions.py [hunk 1]
```diff
@@ -156,8 +184,8 @@ def do(year, format, proj, email, cities):
     # delete directory after zipping:
     shutil.rmtree(tmpdir)
     print('Removed dir %s' % tmpdir)
-
-    # return zip file
-    time.sleep(10)
-    return 'done with %s ! We should now send an email to %s with a link to %s' % (cities, email, zip_name)
+    # send email with a link to download the generated archive:
+    sendmail(email, "Extraction terminée: {0}".format(BASE_URL + '/retrieve/' + uuid))
```
Comment (Vampouille) [inline celery/extractions.py]: Can replaced with : 
```python
 "Extraction terminée: %s/retrieve/%s" % (BASE_URL, uuid)
```

File: celery/extractions.py [hunk 2]
```diff
@@ -117,11 +130,26 @@ def export_schema_to_sql(year, proj, output_dir, conn, pg_connect_string):
...
+        print('Error: unable to send email to %s' % to)
+
+
 @taskmanager.task(name='extraction.do')
 def do(year, format, proj, email, cities):
 
-    extraction_id = 'foncier_{0}{1}{2}_{3}'.format(year, format, proj, do.request.id)
-
+    uuid = do.request.id
+    extraction_id = 'foncier_{0}_{1}_{2}_{3}'.format(year, format, proj, uuid)
```
Comment (Vampouille) [inline celery/extractions.py:151]: Please use fixed format : `extraction_id = 'foncier_{0}{1}{2}_{3}'.format(year, format, proj, do.request.id)` with no spaces between `{0}`, `{1}` and `{2}` ?

File: docker-compose.yml [hunk 3]
```diff
@@ -21,12 +23,14 @@ worker:
     - CELERY_BROKER_URL=redis://redis:6379/0
     - CELERY_RESULT_BACKEND=redis://redis:6379/0
     - PG_CONNECT_STRING=host=database port=5432 user=geoserver dbname=geoserver password=geoserver
+    - LOCAL_SMTP_HOST=smtp
```
Comment (Vampouille) [inline docker-compose.yml:26]: What about smtp port ?

Comment (Vampouille) [reply docker-compose.yml:26]: ok , it is defined as default value

File: celery/extractions.py (hunk 2 above)
Comment (fvanderbiest) [reply celery/extractions.py:151]: nope, it's very moche.

File: foncier/app.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 
 from flask import Flask, render_template, request, Response, g, stream_with_context
-from utils import acces_foncier, extract_cp
+from utils import *
```
Comment (Vampouille) [inline foncier/app.py]: I think it is more readable with complete list of import instead of `*`

Comment (fvanderbiest) [reply foncier/app.py]: ok :-)

File: resources/config.py [hunk 5]
```diff
@@ -2,3 +2,4 @@
 
 DEBUG = False
 LDAP_URI = 'ldap://ldap:389'
+FONCIER_EXTRACTS_DIR = '/extracts'
```
Comment (Vampouille) [inline resources/config.py]: just a missing new line here

**Pull request merged**